staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_type.go

```diff
@@ -387,17 +388,36 @@ func buildSubresourceDefaultVerbTemplates(generateApply bool) map[string]string
 
 func buildDefaultVerbTemplates(generateApply bool) map[string]string {
 	m := map[string]string{
-		"create": `Create(ctx context.Context, $.inputType|private$ *$.inputType|raw$, opts $.CreateOptions|raw$) (*$.resultType|raw$, error)`,
-		"update": `Update(ctx context.Context, $.inputType|private$ *$.inputType|raw$, opts $.UpdateOptions|raw$) (*$.resultType|raw$, error)`,
-		"updateStatus": `// Add a +genclient:noStatus comment above the type to avoid generating UpdateStatus().
-generic.StatusUpdater[*$.inputType|raw$]`,
+		"create":           `Create(ctx context.Context, $.inputType|private$ *$.inputType|raw$, opts $.CreateOptions|raw$) (*$.resultType|raw$, error)`,
```

Comment:
Do we still need this?  If so, can we use `generic.Creator` and `generic.Updater` and so on here ?  I don't quite follow why this commit is a good change, if we end up duplicating code but not fixing the old?

Reply:
The last commit in the sequence is tentative; I wanted to see if it was useful. We still need the separate templates for use in expansions, because of the `adjustTemplate` stuff.

Reply:
My feeling is that the last 2 commit-pairs are retrograde motion - a step too far into DRY.  It forces a consumer of the client to go into the guts of the implementation in order to really understand it, which I think will make docs and things like IDEs worse.

I think we should emit the fully expanded method signatures for types.  Yes it is some duplication, but it seems easier to consume, and that's ultimately what the client is for, right?